README.rst

```diff
@@ -67,3 +67,23 @@ It can then be copied out of the container image, for example into
 ``/path/to/images``::
 
     podman run --volume /path/to/images:/target:Z --rm ironic-python-agent:latest
+
+block-device-yaml
+-----------------
+
+The file ``dib/edpm-partition-uefi/block-device-default.yaml`` is generated by
+the script ``block-device-yaml`` by running::
+
+    ./block-device-yaml --disk-size 5GiB --output dib/edpm-partition-uefi/block-device-default.yaml
+
+This script can also generate a block device description with custom LVM volume
+layouts. For example, to generate a block device layout which replicates the
+TripleO overcloud-hardened-uefi-full.qcow2 which has an extra /srv mount point,
+run the following::
+
+    ./block-device-yaml --disk-size 6GiB \
+        --volumes lv_root=:lv_tmp=240:lv_var=952:lv_log=240:lv_audit=192:lv_home=240:lv_srv=48 \
```

Comment:
More detail into what the options and the syntax is might be helpful.

Reply:
+1, and we may want to use `ArgumentDefaultsHelpFormatter` instead of `RawDescriptionHelpFormatter` - to get a more useful output from `--help` ?


https://docs.python.org/3/library/argparse.html#argparse.ArgumentDefaultsHelpFormatter

Reply:
I've used multiple inheritance to add ArgumentDefaultsHelpFormatter